core/utils.py

```diff
@@ -34,22 +35,24 @@ def get_pypi_url(title: str, timeout: float = 1.0):
             return None
 
 
-STATUS_CHOICES = (
-    (0, "Unknown"),
-    (1, "Development Status :: 1 - Planning"),
-    (2, "Development Status :: 2 - Pre-Alpha"),
-    (3, "Development Status :: 3 - Alpha"),
-    (4, "Development Status :: 4 - Beta"),
-    (5, "Development Status :: 5 - Production/Stable"),
-    (6, "Development Status :: 6 - Mature"),
-    (7, "Development Status :: 7 - Inactive"),
-)
+class PackageStatus(models.IntegerChoices):
+    UNKNOWN = 0, "Unknown"
+    PLANNING = 1, "Development Status :: 1 - Planning"
+    PRE_ALPHA = 2, "Development Status :: 2 - Pre-Alpha"
+    ALPHA = 3, "Development Status :: 3 - Alpha"
+    BETA = 4, "Development Status :: 4 - Beta"
+    STABLE = 5, "Development Status :: 5 - Production/Stable"
+    MATURE = 6, "Development Status :: 6 - Mature"
+    INACTIVE = 7, "Development Status :: 7 - Inactive"
+
+
+STATUS_CHOICES = PackageStatus.choices
```

Comment:
Since this is only used in one spot, the following function, I would just remove this and use `PackageStatus.choices` directly.